config/environments/development.rb

```diff
@@ -24,15 +39,13 @@
   # number of complex assets.
   config.assets.debug = true
 
-  # Asset digests allow you to set far-future HTTP expiration dates on all assets,
-  # yet still be able to expire them through the digest params.
-  config.assets.digest = true
```

Comment:
Is this no longer a valid option in Rails 5? If so I think it'd be good to explain that in the commit note.